dmoj/cptbox/tracer.py

```diff
@@ -195,7 +195,9 @@ def wait(self):
         self._died.wait()
         if not self.was_initialized:
             if self.returncode == 203:
-                raise RuntimeError('failed to set up seccomp policy')
+                out = self.stdout.read() if self.stdout else '(nil)'
+                err = self.stderr.read() if self.stderr else '(nil)'
+                raise RuntimeError('failed to set up seccomp policy, stdout="%s", stderr="%s"' % (out, err))
```

Comment:
This and the .docker/entry change won't be included in the final PR, just need these for now to test.